### PR TITLE
Enforce correct protobuf version

### PR DESCRIPTION
**Pull request by fbeutin-ledger**

### CODE REVIEW

**Comment by xchapron-ledger** [comment]
I rather regenerate the `ledgerwallet/proto/LedgerHSMServer_pb2.py` file we an updated protobuf version if we can...

**Review by xchapron-ledger — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -17,12 +17,16 @@ jobs:
...
       run: pip install .
+
     - name: Install test dependencies
-      run: pip install nose coverage
-    - name: Running unit tests
-      run: nosetests --with-coverage --cover-package ledgerwallet --cover-xml tests/unit/
+      run: pip install -r tests/unit/requirements.txt
+
+    - name: Run unit tests
+      run: pytest --cov ledgerwallet --cov-report=xml tests/unit/
```
Comment (mlafon-ledger) [inline .github/workflows/ci.yml:28]: Maybe we could set `--cov-report term --cov-report xml` to have the coverage directly in the action view and also in the xml file

Comment (fbeutin-ledger) [reply .github/workflows/ci.yml:28]: Done

**Comment by fbeutin-ledger** [comment]
Approved by @mlafon-ledger in comment

**Pull request merged**